crates/datasources/src/postgres/mod.rs

```diff
@@ -604,13 +633,22 @@ impl ExecutionPlan for BinaryCopyExec {
         _partition: usize,
         _context: Arc<TaskContext>,
     ) -> DatafusionResult<SendableRecordBatchStream> {
-        let stream = ChunkStream {
-            state: StreamState::Idle,
-            types: self.pg_types.clone(),
-            opener: self.opener.clone(),
-            arrow_schema: self.arrow_schema.clone(),
-        };
-        Ok(Box::pin(stream))
+        todo!()
+        // Box::pin(async move {
+        //     let opener = StreamOpener::new(
+        //         self.query.clone(),
+        //         self.connection_string.clone(),
+        //         self.tunnel.clone(),
+        //     )
+        //     .await;
+        //     let stream = ChunkStream {
+        //         state: StreamState::Idle,
+        //         types: self.pg_types.clone(),
+        //         opener: opener?,
+        //         arrow_schema: self.arrow_schema.clone(),
+        //     };
+        //     stream
+        // })
     }
```

Comment:
@scsmithr I've been struggling with figuring out how to implement the `execute` in a way that doesnt rely on `async`. 

the `tokio_postgres::Client` needs to be created inside the `execute`, but it relies on a tokio runtime. But since we need this serializeable, we can't have `BinaryCopyExec` hold a connection handle, or a runtime handle. 

We _could_ just block on this since `execute` is already sync. But wasn't sure if there was maybe a better way.